src/frontend/screens/ObservationsList/index.tsx

```diff
@@ -63,11 +68,19 @@ export const ObservationsList: React.FC<
 
   return (
     <View style={styles.container} testID="observationsListView">
+      {isLoading ? (
+        <Loading />
+      ) : data && data.length <= 1 ? (
+        <NoProjectWarning style={{margin: 20}} />
```

Comment:
this `data.length <= 1` is the temporary solution for determining whether the user is on the default project. Since there is not support in the front end for multi-projects, the user is always going to be in the latest project they made. So if they have only 1 project, we know they are in the default project.

Reply:
This isn't ideal but it's fine. I'd probably file an issue for this and refer to this line so we don't forget to fix it once we add multi-project support.

I'd also add _some_ explanation for what this is doing. I'd probably use a variable like `isUserOnDefaultProject`.